### PR TITLE
Fix wrong encryption message for block and file storage

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Resolve: bz#2038873

Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/security-and-network-step/encryption.tsx [hunk 1]
```diff
@@ -204,9 +204,7 @@ export const Encryption: React.FC<EncryptionProps> = ({
   };
 
   const description = !isMCG
-    ? t(
-        'ceph-storage-plugin~Data encryption for block and file storage. MultiCloud Object Gateway is always encrypted.',
-      )
+    ? t('ceph-storage-plugin~Data encryption for block and file storage.')
```
Comment (GowthamShanmugam) [inline frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/security-and-network-step/encryption.tsx:207]: This is already migrated to odf-console, fix has to be in odf-console only. 

cc: @SanjalKatiyar

Comment (SanjalKatiyar) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/security-and-network-step/encryption.tsx:207]: yes @GowthamShanmugam u r right...

also, @TimothyAsirJeyasing plz so thr the bug comments before raising the PR https://bugzilla.redhat.com/show_bug.cgi?id=2038873#c10 
 I don't think we have done what is being asked...

Comment (TimothyAsirJeyasing) [reply frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/security-and-network-step/encryption.tsx:207]: @SanjalKatiyar Do you mean we need a text like
ceph-storage-plugin~Data encryption for block and file storage. MultiCloud Object Gateway supports encryption for objects by default

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh , @SanjalKatiyar Please review

**Comment by SanjalKatiyar** [comment]
/close
not needed in this repo.

**Pull request closed without merging**